lib/galaxy/tools/parameters/basic.py

```diff
@@ -84,6 +84,13 @@ class ToolParameter(Dictifiable):
     Describes a parameter accepted by a tool. This is just a simple stub at the
     moment but in the future should encapsulate more complex parameters (lists
     of valid choices, validation logic, ...)
+
+    >>> from galaxy.util.bunch import Bunch
+    >>> trans = Bunch(app=None)
+    >>> p = TextToolParameter(None, XML('<param argument="--parameter-name" type="text" value="default" />'))
```

Comment:
Given this is a doctest for `ToolParameter`, I'd use:
```suggestion
    >>> p = ToolParameter(None, XML('<param argument="--parameter-name" type="text" value="default" />'))
```
and update the assert below as needed.